### PR TITLE
Add Dark mode

### PR DESCRIPTION
**Pull request by amochuko**
The PR enable a Dark theme of the web app

## Related GitHub Issue

Update related to https://github.com/nation3/citizen-app/issues/292

## How Has This Been Tested?

- [x] Status checks pass (lint, build, test)
- [x] Works on Sepolia preview deployment
- [x] Works on Mainnet preview deployment

## Are There Admin Tasks?
No

### CODE REVIEW

**Comment by amochuko** [comment]
WIP with remaining two items:

1. The logo 
2. Hamburger icon (Nav bar)

Kindly take a look over and let me your thoughts.
Thanks

**Comment by amochuko** [comment]
> WIP with remaining two items:
> 
> 1. The logo
> 2. Hamburger icon (Nav bar)
> 
> Kindly take a look over and let me your thoughts. Thanks

I got item 2 done; and created a placeholder for item 1 which I intend to fill with a logo that would fit a dark mode.

**Review by aahna-ashina — CHANGES_REQUESTED** [review]
@amochuko When I hover a menu item in light mode, the item's background color changes.  Can we keep this functionality in dark mode?

> 

>

**Comment by amochuko** [comment]
> @amochuko When I hover a menu item in light mode, the item's background color changes. Can we keep this functionality in dark mode?
> 
> > 
> 
> > 

Sure, we can. I'll go rectify it.

**Comment by amochuko** [comment]
> @amochuko When I hover a menu item in light mode, the item's background color changes. Can we keep this functionality in dark mode?
> 
> > 
> 
> > 

I've made a fix.

File: ui/components/HomeCard.tsx [hunk 1]
```diff
@@ -10,10 +10,12 @@ export default function HomeCard({
 }: any) {
   return (
     <Link href={href} passHref>
-      <div className="card bg-base-100 shadow-md transition ease-in-out hover:-translate-y-1 cursor-pointer">
-        <div className="card-body items-stretch items-center ">
+      <div className="card bg-base-100 shadow-md transition ease-in-out hover:-translate-y-1 cursor-pointer dark:bg-slate-600">
```
Comment (aahna-ashina) [inline ui/components/HomeCard.tsx:13]: @amochuko Try reducing the brightness of the `HomeCard`, for less contrast.  For example slate-800 or something closer to the background color.

> 

The contrast between the card and the background should be similar as the one in light mode:

>

Comment (amochuko) [reply ui/components/HomeCard.tsx:13]: Using `slate-700` would be best to reducing the brightness of the HomeCard, for less contrast as `slate-800` is the actual background shade.

With regards to uniformity, I should also reflect that for the MainCard component?

Comment (aahna-ashina) [reply ui/components/HomeCard.tsx]: Yes, that looks better 👍

Comment (aahna-ashina) [reply ui/components/HomeCard.tsx]: > With regards to uniformity, I should also reflect that for the MainCard component?
> 

Yes, please.

Comment (amochuko) [reply ui/components/HomeCard.tsx]: > > With regards to uniformity, I should also reflect that for the MainCard component?
> 
> Yes, please.

I've did so, and made a push earlier.

**Comment by aahna-ashina** [comment]
Thank you, @amochuko.  Can you also change the `alert` div at https://citizen-app-mainnet-git-fork-amochuko-dark-mode-nation3dao.vercel.app/lock into dark theme?

>

**Comment by amochuko** [comment]
> Thank you, @amochuko. Can you also change the `alert` div at https://citizen-app-mainnet-git-fork-amochuko-dark-mode-nation3dao.vercel.app/lock into dark theme?
> 
> > 

I've updated the alert on Dark mode and ensured keeping it still noticeable.

**Comment by aahna-ashina** [comment]
> I've updated the alert on Dark mode and ensured keeping it still noticeable.

@amochuko I don't see any dark theme added with your latest commit.  Am I looking at the correct preview URL? - https://citizen-app-mainnet-git-fork-amochuko-dark-mode-nation3dao.vercel.app/

>

**Review by aahna-ashina — APPROVED** [review]

**Pull request merged**